.github/workflows/acceptance_tests_gks.yaml

```diff
@@ -0,0 +1,81 @@
+name: Acceptance Tests (GKS)
+
+on:
+  workflow_dispatch: 
+    inputs:
+      region: 
+        description: The GKS region 
+        default: us-east1
+      clusterVersion:
+        description: The GKS cluster version
+        default: 1.21
+      workersCount:
+        description: The number of cluster nodes to provision 
+        default: 2
+      nodeMachineType:
+        description: The type of GKE instance to use for cluster nodes
+        default: e2-standard-2
+      runTests:
+        description: The regex passed to the -run option of `go test`
+        default: ".*"
+      terraformVersion:
+        description: Terraform version 
+        default: 1.1.8
+
+env:
+  GOOGLE_CREDENTIALS: '${{ secrets.GKE_SA_KEY }}'
+  GOOGLE_PROJECT: '${{ secrets.GOOGLE_PROJECT }}'
+  GOOGLE_REGION: ${{ github.event.inputs.region }}
+
+jobs:
+  acceptance_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Read go-version
+        id: go-version
+        uses: juliangruber/read-file-action@v1
+        with:
+          path: ./.go-version
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ steps.go-version.outputs.content }}
+      - name: Authenticate to Google Cloud
+        uses: google-github-actions/setup-gcloud@v0
```

Comment:
I think you need to run their [auth](https://github.com/google-github-actions/auth#authenticating-via-service-account-key-json-1) action before you run the `setup-glcoud` action.